### PR TITLE
Update mixer.adapter.model.v1beta1.Value

### PR DESCRIPTION
**Pull request by mandarjog**
1. Update mixer.adapter.model.v1beta1.Value to be oneof other mixer type.
2. Add concrete values to all Mixer types.

### CODE REVIEW

File: mixer/adapter/model/v1beta1/type.proto [hunk 1]
```diff
@@ -18,26 +18,74 @@ package istio.mixer.adapter.model.v1beta1;
...
+        // Used for values of type TIMESTAMP
+        TimeStamp timestamp_value = 6;
+
+        // Used for values of type DURATION
+        Duration duration_value = 7;
+
+        // Used for EmailAddress
+        EmailAddress email_address_value = 8;
+
+        // Used for DNSName
```
Comment (guptasu) [inline mixer/adapter/model/v1beta1/type.proto]: for parallel statements `Used for values of type DNSName`

File: mixer/adapter/model/v1beta1/type.proto [hunk 2]
```diff
@@ -18,26 +18,74 @@ package istio.mixer.adapter.model.v1beta1;
...
 
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
 // Value is used inside templates for fields that have dynamic types. The actual datatype
 // of the field depends on the datatype of the expression used in the operator configuration.
-message Value {}
+message Value {
+    oneof value {
+        // Used for values of type STRING, DNS_NAME, EMAIL_ADDRESS, and URI
```
Comment (guptasu) [inline mixer/adapter/model/v1beta1/type.proto]: Why is this not just `string` ?

File: mixer/adapter/model/v1beta1/type.proto [hunk 3]
```diff
@@ -18,26 +18,74 @@ package istio.mixer.adapter.model.v1beta1;
...
+        // Used for values of type DURATION
+        Duration duration_value = 7;
+
+        // Used for EmailAddress
+        EmailAddress email_address_value = 8;
+
+        // Used for DNSName
+        DNSName dns_name_value = 9;
+
+        // Used for Uri
```
Comment (guptasu) [inline mixer/adapter/model/v1beta1/type.proto]: for parallel statements Used for values of type URI

File: mixer/adapter/model/v1beta1/type.proto [hunk 4]
```diff
@@ -18,26 +18,74 @@ package istio.mixer.adapter.model.v1beta1;
...
 // Duration is used inside templates for fields that are of ValueType "DURATION"
-message Duration {}
+message Duration {
+    google.protobuf.Duration value = 1;
+}
 
 // TimeStamp is used inside templates for fields that are of ValueType "TIMESTAMP"
-message TimeStamp {}
+message TimeStamp {
+    google.protobuf.Timestamp value = 1;
```
Comment (guptasu) [inline mixer/adapter/model/v1beta1/type.proto]: missing comment. Ditto for other places too

**Comment by mandarjog** [comment]
updated comments,PTAL.

**Pull request merged**